screens/auth/Login.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Text, View, Image } from 'react-native';
+import Input from '../../components/InputField';
+import styles from '../../Style';
+import Button from '../../components/Button';
+
+const Login = () => {
+  return (
+    <View style={styles.loginContainer}>
+      <Image
+        source={{ uri: 'https://upload.wikimedia.org/wikipedia/commons/thumb/4/4f/Twitter-logo.svg/1245px-Twitter-logo.svg.png' }}
+        style={styles.twitterLogo}
+      />
+      <Text style={styles.logintoTwitterStyle}>Log in to Twitter</Text>
+
+      <Input
+        placeholder={'Username'}
+        onChangeText={ (text) => {
+          throw new Error('Function not implemented.');
```

Comment:
remove this throw error, save the value on the states instead and use it as a value